src/routes/authRoutes.ts

```
@@ -212,8 +213,9 @@ import {
 const router: ReturnType<typeof Router> = Router();
 
 router.post("/signup", authRateLimiter, postSignup);
-router.post("/signin", authRateLimiter, postSignin);
-router.post("/signin/verify-2fa", authRateLimiter, postVerify2fa);
+// #269: twoFaRateLimiter adds per-user/IP keyed limiting on top of the IP-only authRateLimiter
+router.post("/signin", authRateLimiter, twoFaRateLimiter, postSignin);
+router.post("/signin/verify-2fa", authRateLimiter, twoFaRateLimiter, postVerify2fa);
```

ESLint

[error] Replace `"/signin/verify-2fa",·authRateLimiter,·twoFaRateLimiter,·postVerify2fa` with `⏎··"/signin/verify-2fa",⏎··authRateLimiter,⏎··twoFaRateLimiter,⏎··postVerify2fa,⏎`

(prettier/prettier)